python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -32,7 +32,10 @@
 import sys
 sys.stdout.reconfigure(encoding='utf-8')
 
-from ipex_llm.utils import BenchmarkWrapper
+try:
+    from ipex_llm.utils import BenchmarkWrapper
+except:
+    pass
```

Comment:
Should we fix the `except` logic for npu-related tesg api?

Reply:
> Should we only use the `except` logic for npu-related test_api?

But it's not quite convenient to add test API related check here. This is only a work around, we could remove it after support.